src/components/EventForm/EventForm.js

```diff
@@ -0,0 +1,142 @@
+import React from "react"
+import { Button, TextField } from "@material-ui/core";
+import Grid from '@material-ui/core/Grid';
+import RadioGroupButtons from './formParts/RadioGroupButtons';
+
+const textFieldLabelFontSize = "17px";
+
+export default function Form(props) {
+    const {
+        values: { email, fname, lname, id },
+        errors,
+        touched,
+        handleSubmit,
+        handleChange,
+        setFieldTouched
+    } = props;
+
+    console.log(props);
+
+    const change = (name, e) => {
+        e.persist();
+        handleChange(e);
+        setFieldTouched(name, true, false);
+    };
+
+    return (
+        <form
+            onSubmit={handleSubmit}
+        >
+
+            <h4 style={{ color: 'red' }}>
+                *Required
+            </h4>
+            <Grid container spacing={3}>
+
+                <Grid item xs={12} sm={6}>
+                    <TextField
+                        label="First Name*"
+                        autoComplete="given-name"
+                        helperText={touched.firstname ? errors.firstname : ""}
+                        error={touched.firstname && Boolean(errors.firstname)}
+                        id="fname"
```

Comment:
do you know if the `id` is actually used anywhere?

Reply:
Yea it's needed for the wrapper to work properly when it gets the inputs in submitValues